main/webapp/modules/core/scripts/util/dialog.js

```diff
@@ -42,6 +42,13 @@ var escapeKey = function(event) {
   }
 }
 
+/**
+ * Show a dialog.
+ * @public
+ * @param {jQuery} elmt The dialog element to show.
+ * @param {function} onCancel The function to call when the dialog is dismissed.
```

Comment:
Since this callback is passed through to another function (and I can't find any uses in the project), I think it would be helpful to further describe this parameter using the [callback annotation](https://jsdoc.app/tags-callback). What do you think?

Reply:
Either that or by writing it like this: `@param {function(int)}`, the callback annotation is a little bit verbose for my taste as it needs an additional block, but it's a matter of taste and I'm happy either way.